src/main/java/mods/eln/Eln.java

```diff
@@ -5515,6 +5546,13 @@ void recipeHeatFurnace() {
 				Character.valueOf('i'), findItemStack("Copper Thermal Cable"),
 				Character.valueOf('I'), findItemStack("Combustion Chamber"));
 
+		addRecipe(findItemStack("Fuel Heat Furnace"),
```

Comment:
The smallest fuel burner has a power of 5 kW, the heat furnace has 1-2kW of heat power (depends the number of combustion chambers), so the smallest burner has more than twice the power of the simple heat furnace. Imo this is ok.